Parsedown.php

```diff
@@ -1453,8 +1458,19 @@ protected function elements(array $Elements)
         return $markup;
     }
 
+    #
+    # - [x] AND - [ ]
+    protected function checkbox(&$text,&$attributes)
+    {
+        if(strpos($text,'[x]')!==false||strpos($text,'[ ]')!==false){
+            $attributes = array("style"=>"list-style: none;");
```

Comment:
Also Parsedown has no precedent of inline styles thus far – this could break in applications that deploy a CSP, perhaps a non `li` could be used instead for the entire structure?